src/App.js

```diff
@@ -1,15 +1,13 @@
-import React from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
-import NavMenu from "./components/NavMenu";
-import Welcome from "./components/Welcome";
-import RSVP from "./components/RSVP";
+import React from "react";
+// import RSVPform from "./components/RSVPform";
```

Comment:
Comments here

Reply:
was commenting that component off to only render FindInvite.